.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Prepare Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to release
+        required: true
+      force:
+        description: Force a release even when there are release-blockers (optional)
+        required: false
+      merge_target:
+        description: Target branch to merge into. Uses the default branch as a fallback (optional)
```

Comment:
let's default this to `sentry-v1` I guess?

Reply:
Turns out, `sentry-v1` is already the default branch, so it should select this one anyway already if no other branch is explicitly specified. I'll just adjust the label on the input field.